src/clr-icons/shapes/chart-shapes.ts

```diff
@@ -222,24 +222,26 @@ export const ClrShapeBellCurve = clrIconSVG(
   `<path d="M33,29H3A1,1,0,1,1,3,27H33A1,1,0,1,1,33,29Z" class="clr-i-outline clr-i-outline-path-1" /><path d="M33,25h-.62a8.11,8.11,0,0,1-8-6.67C23.62,14.44,21.89,7.94,18,7.94s-5.69,6.51-6.38,10.39a8.11,8.11,0,0,1-8,6.65H3a1,1,0,1,1,0-2h.6A6.11,6.11,0,0,0,9.6,18c1.41-7.88,4.3-12,8.35-12s6.93,4.16,8.33,12a6.11,6.11,0,0,0,6,5H33a1,1,0,0,1,0,2Z" class="clr-i-outline clr-i-outline-path-2" />`
 );
 
-export const ChartShapes: any = {
+export const ChartShapes = {
   'axis-chart': ClrShapeAxisChart,
   'bar-chart': ClrShapeBarChart,
+  'bell-curve': ClrShapeBellCurve,
+  'box-plot': ClrShapeBoxPlot,
   'bubble-chart': ClrShapeBubbleChart,
   'cloud-chart': ClrShapeCloudChart,
   'curve-chart': ClrShapeCurveChart,
   'grid-chart': ClrShapeGridChart,
+  'heat-map': ClrShapeHeatMap,
   'line-chart': ClrShapeLineChart,
   'pie-chart': ClrShapePieChart,
-  'tick-chart': ClrShapeTickChart,
   'scatter-plot': ClrShapeScatterPlot,
-  'box-plot': ClrShapeBoxPlot,
-  'heat-map': ClrShapeHeatMap,
-  'bell-curve': ClrShapeBellCurve,
+  'tick-chart': ClrShapeTickChart,
+  /** Alias for: line-chart */ 'analytics': ClrShapeLineChart,
```

Comment:
Oops, I overlooked this one. This should point to the reference of the shape. That's why it was like this:

```
Object.defineProperty(ChartShapes, 'analytics', descriptorConfig(ChartShapes['line-chart']));
```

Otherwise, it will dump its SVG markup again in the published .js file that will dramatically increase the file size.

Reply:
The issue with using `Object.defineProperty` is that you loss TypeScript inferance, because it can't predict what are you doing in there. The effect of the following 3 examples is exactly the same:
```js
const foo = "foo";
const bar = "bar";

const example1 = { foo: foo, bar: bar, baz: bar };
// { foo: "foo", bar: "bar", baz: "bar" }

const example2 = { foo: foo, bar: bar };
example2.baz = bar;
// { foo: "foo", bar: "bar", baz: "bar" }

const example3 = { foo: foo, bar: bar };
Object.defineProperty(example3, 'baz', descriptorConfig(example3.bar));
// { foo: "foo", bar: "bar", baz: "bar" }
```

I even took a look at the minified code in `@clr/icons/shapes/all-shapes.min.js`, and this doesn't have any kind of performance effect.

Reply:
> The effect of the following 3 examples is exactly the same

example 3 is different as it doesn't assign `baz` **the value of bar**, but the reference of  bar.  It's the same as using object literal getter: 
```
const example3 = {
   foo: foo, 
   bar: bar,
   get baz() {
      return this.bar;
   }
};
```
As icons have a large svg templates, we should only reference to the existing shapes in the set, instead of dumping the same markup for the aliases, which would result in larger file `.min.js` sizes.

Also, I just remember why we couldn't just use object literal `getter` at the first place and had to resort to `Object.defineProperty`. It was because UglifyJS had a bug/problem that doesn't recognize getter properties with `-` in it and creates runtime errors when users try to use them.
So I am afraid we couldn't use your changes on these aliases' part. I am open to your suggestions though.

Reply:
I tested the getter approach with UglifyJS and it works as expected:
```js
const foo = "foo";
const bar = "bar";
const example = {
	foo: foo,
	bar: bar,
	get "this-works"() {
		return this.bar;
	},
	get thisAlsoWorks() {
		return this.bar;
	}
};
```
Returns:
```js
const foo="foo",bar="bar",example={foo:foo,bar:bar,get"this-works"(){return this.bar},get thisAlsoWorks(){return this.bar}};```

I feel is the best one, because it complies with the minifying requirements and also with the typings for TS.